### DISCUSSION TITLE
vueform uploadTempFile error

### DISCUSSION THREAD

**Discussion by 490727178** [vueform/vueform#317] [Questions]
![image](https://github.com/user-attachments/assets/b681c7fd-4a4e-41e4-aef1-d29e2365344f)
![image](https://github.com/user-attachments/assets/d3c88f9b-3c0e-420a-a51e-f844fd62ba84)
![image](https://github.com/user-attachments/assets/d0d33a0c-2efd-4c56-8571-f2c8c1ac2971)

**Comment by adamberecz** [comment]
Hey, this was in fact a bug so thanks for catching it.

A fix is released in `1.10.6`
